joint_teapot/workers/canvas.py

```diff
@@ -177,6 +177,9 @@ def upload_assignment_grades(
             )
         logger.info(f"Canvas assginemnt {assignment} grades upload succeed")
 
+    def get_student_uniqnames(self) -> List[str]:
```

Comment:
is it used?

Reply:
I removed the code calling it. This was used to import students from canvas to mm directly, but it didn't work well. Shall I remove this as well?

Reply:
yes